worker/src/RTC/RtpDictionaries/RtpCodecParameters.cpp

```diff
@@ -24,9 +24,10 @@ namespace RTC
 		this->clockRate = data->clockRate();
 
 		// channels is optional.
-		if (data->channels().has_value())
+		auto channels = data->channels();
+		if (channels.has_value())
```

Comment:
What is going on in this PR? We always add blank lines after variable declaration and before `if` conditions and before `return`.


```suggestion
		auto channels = data->channels();

		if (channels.has_value())
```

Reply:
You're right, I was unconsciously doing that within this PR because such variable `channels` only concerns the conditional block and tightening it to the condition statement made sense to me.

For such cases like this one C++17 exposes the [if statements](https://en.cppreference.com/w/cpp/language/if.html) with initializers which would lead to this outcome:

```c++
// `channels` variable scope is limited to the condition block.
if (auto channels = data->channels(); channels.has_value())
{
  this->channels = channels.value();
}
```

This also allow us to reuse the `channels` name for variables outside the condition.

I'm fine with this for this cases. WDYT @ibc?

Reply:
Fine